src/UI/Header/index.js

```diff
@@ -1,23 +1,12 @@
-import React from 'react';
-import { Link } from 'react-router-dom'
+import React from "react";
+import HeaderNav from "./Nav";
 
-const Header = () => {
-  return(
+const HeaderContainer = () => {
+  return (
     <header>
-      <nav>
-        <Link to="/">Home</Link>
-        {' '}
-        <Link to="/about">About</Link>
-        {' '}
-        <Link to="/contact">Contact</Link>
-        {' '}
-        <Link to="/services">Towing Services</Link>
-      </nav>
-      <figure>
```

Comment:
I moved it into `src/repeats/Header/Image.js` since `<figure>` image is different from the group of `<Links>`. I wanted to enforce SRP, do you think it was done well?

Reply:
Sorry, absolutely its in here (398cd4)[https://github.com/imagineTech/united_trucking_service/pull/3/commits/398cd4eac739db51970de66f97ff2716abf2ec2a]
Its good. :+1: